packages/react/src/components/AccountSettings/ChangePassword/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface ChangePasswordProps {
```

Comment:
More `ChangePassword` types will be added to this file in later PRs, like validator types.